src/tss2-fapi/ifapi_policy_callbacks.c

```diff
@@ -1366,9 +1366,14 @@ ifapi_exec_auth_nv_policy(
 
     switch (cb_ctx->cb_state) {
         statecase(cb_ctx->cb_state, POL_CB_EXECUTE_INIT)
-            /* Search a NV object with a certain NV indext stored in nv_public. */
+            /*
+             * Search a NV object with a certain NV indext stored in nv_public.
+             * TODO Consider modifying internal APIs to use TPMS_NV_INDEX over TPM2B variant.
```

Comment:
Modifying this keystore search really starts to bleed into other areas of the codebase, might be better just to leave it alone, but I could modify if it needed.

Reply:
> Modifying this keystore search really starts to bleed into other areas of the codebase, might be better just to leave it alone, but I could modify if it needed.

@williamcroberts I would agree to leave it alone. Also in the JSON policy spec TPM2B_PUBLIC is used for nvPublic in TPMS_POLICYAUTHORIZENV.